frontend/src/data/utils/Fips.ts

```diff
@@ -2,6 +2,14 @@ export const USA_DISPLAY_NAME = "United States";
 // Fake FIPS code used to represent totals in USA for convenience
 export const USA_FIPS = "00";
 
+export const TERRITORY_CODES = [
+  /*American Samoa-*/ "60",
+  /*Guam-*/ "66",
+  /*Northern Mariana Islands-*/ "60",
```

Comment:
Is this right? You seem to have "60" twice

Reply:
I think Northern Marian Islands should be 69

Reply:
oo good catch thanks!